dashboard/app/models/backpack.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: backpacks
+#
+#  id             :bigint           not null, primary key
+#  user_id        :integer          not null
+#  storage_app_id :integer          not null
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#
+# Indexes
+#
+#  index_backpacks_on_user_id  (user_id)
```

Comment:
Confirming we won't need to do the reverse lookup (go from storage_app_id -> user)?

Reply:
no, should only need to lookup the backpack for a user, not vice versa

Reply:
When purging a user, we need to be able to find all of the content they’ve created and delete all of their project S3 Objects.